samples/test/automlTablesPredict.v1beta1.test.js

```diff
@@ -15,74 +15,116 @@
 
 'use strict';
 
-const {assert} = require('chai');
+const {assert, expect} = require('chai');
 const {describe, it} = require('mocha');
-const {execSync} = require('child_process');
+const proxyquire = require('proxyquire');
 
 /** Tests for AutoML Tables "Prediction API" sample. */
 
-const cmdPredict = 'node automlTablesPrediction.js';
-
-// // TODO(developer): Before running the test cases, set the environment
-// variables PROJECT_ID, REGION_NAME and
-// change modelId, gcsInputUri, gcsOutputUriPrefix, bqInputUri and
-// bqOutputUriPrefix.
-//const projectId = process.env.PROJECT_ID;
-//const computeRegion = process.env.REGION_NAME;
-const modelId = 'TBL3613734080685801472';
-const filePath = './resource/predictTest.csv';
-const gcsInputUri = 'gs://automl-tables/input/test.csv';
-const gcsOutputUriPrefix = 'gs://automl-tables/output';
+const projectId = process.env.AUTOML_PROJECT_ID;
```

Comment:
We really like to avoid specialized env vars.  Why can't this use the project Id that the code is running under?

Reply:
I am leaning toward to centralized project idea
https://docs.google.com/document/d/1-E7zTNqBm9ex7XIOhzMHCupwKWieyMKgAVwrRK5JTVY/edit?ts=5efa4c76

I would like to keep this consistent like java and python. let me know what you think?

Reply:
Ah, if there are good reasons to have it's own project, so be it.  @bcoe is this the way we manage other projects where the samples need to run under a different project?